configure.ac

```diff
@@ -147,7 +147,7 @@ AC_ARG_WITH(kernel-dir,
   [AS_HELP_STRING([--with-kernel-dir=DIR], [path to linux kernel source directory])],
   [AS_HELP_STRING([kernel_src_path="$withval"],], [[kernel_src_path=""])])
 AC_ARG_ENABLE(fwmark,
-  [AS_HELP_STRING([--disable-fwmark], [compile without SO_MARK support])])
+  [AS_HELP_STRING([--enable-fwmark], [compile with SO_MARK support])])
```

Comment:
I don't think this is correct; fwmark should be enabled by default if LVS is enabled.